library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/items/SingleLineItem.java

```diff
@@ -109,29 +109,17 @@ public int getLayoutRes() {
     public void bindView(ViewHolder holder, List payloads) {
         super.bindView(holder, payloads);
         mName.applyTo(holder.name);
-        if (mAvatar != null) {
-            if (holder.avatar.getVisibility() == View.GONE) {
-                holder.avatar.setVisibility(View.VISIBLE);
-            }
-            mAvatar.applyTo(holder.avatar);
-        } else if (holder.avatar.getVisibility() == View.VISIBLE) {
-            holder.avatar.setVisibility(View.GONE);
-        }
-        if (mIcon != null) {
-            if (holder.icon.getVisibility() == View.GONE) {
-                holder.icon.setVisibility(View.VISIBLE);
-            }
-            mIcon.applyTo(holder.icon);
-        } else if (holder.icon.getVisibility() == View.VISIBLE) {
-            holder.icon.setVisibility(View.GONE);
-        }
+        ImageHolder.applyToOrSetInvisible(mAvatar, holder.avatar);
+        ImageHolder.applyToOrSetInvisible(mIcon, holder.icon);
     }
 
     @Override
     public void unbindView(ViewHolder holder) {
         holder.name.setText(null);
         holder.avatar.setImageDrawable(null);
+        holder.avatar.setVisibility(View.VISIBLE);
```

Comment:
Because i reset the views to the default state. Thats the state the view had after the view got inflated from the layout.